admin/python/lsst/qserv/admin/nodeMgmt.py

```diff
@@ -123,11 +125,11 @@ def selectDict(self, state=None, nodeType=None):
         """
 
         # if state is a string make a list out of it
-        if isinstance(state, types.StringTypes):
+        if isinstance(state, str):
             state = [state]
 
         # if nodeType is a string make a list out of it
-        if isinstance(nodeType, types.StringTypes):
+        if isinstance(nodeType, str):
```

Comment:
Correct, did the linter complain about this? This change is something I know `futurize` does but I didn't think you were doing the python 3 port on this ticket. I think you should use `basestring` for now until you drop python2.